src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java

```diff
@@ -169,11 +161,13 @@ public void setUp() {
         objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
         objectMapper.registerModule(new ContingencyJsonModule());
 
-        cleanDB();
+        scriptContingencyListRepository.deleteAll();
+        formContingencyListRepository.deleteAll();
+        idBasedContingencyListRepository.deleteAll();
```

Comment:
If you clean the DB twice it's nicer to have a method, then you're you don't forget to clean some stuff in one call.

Reply:
huh? we clean only once at the end to not pollute the database for others tests.

Reply:
No, the call to these 3 deletes is done twice, once in setUp and once in tearDown, so it's better if we're sure we call the same 3 deletes in both cases. For that, you can keep the cleanDb() method that was already present.